docs/source/index.rst

```diff
@@ -55,20 +55,64 @@ The QCArchive project's primary support comes from `The Molecular Sciences Softw
 
 ========
 
-Table of Contents
-=================
+Index
+-----
+
+
+**Getting Started**
+
+* :doc:`install`
+* :doc:`community`
+* :doc:`roadmap`
 
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 1
+   :hidden:
    :caption: Getting Started
 
    install
    community
    roadmap
 
+**Portal Documentation**
+
+Portal is the primary user interface to a Fractal server instance.
```

Comment:
I recc. defining a `|qcf|` and `|qcfs|` for "QCFractal" and "QCFractal's" in conf.py and using it everywhere. Shorter to type (unlike `|PSIfour|`) and ensures uniformity throughout docs.